pkg/storage/sqlite/sqlite.go

```diff
@@ -35,6 +35,12 @@ func startTrace(ctx context.Context, name string) (context.Context, trace.Span)
 	return tracer.Start(ctx, "sqlite."+name)
 }
 
+var tupleColumns = []string{
+	"store", "object_type", "object_id", "relation",
+	"user_object_type", "user_object_id", "user_relation", /* TODO: "user_type"?, what is the difference */
```

Comment:
user_type is extracted from user_relation, and we don't select or scan it here because TupleRecord doesn't have a field for it. It is used in postgres and sqlite (but not mysql) to create partial indexes and (hopefully) speed up selecting from the tuple table.